cypress/integration/Train/MyNameIs.spec.js

```diff
@@ -60,9 +60,9 @@ describe('My Name Is - Train Dialog', () => {
       // Bug 1956: Replaced Entity Indicator has been lost when training, can still be seen when editing a training
       // This test will fail once the bug has been fixed.
       // Also uncomment the 3rd line of code.
```

Comment:
Does comment need updating?  Seems like you uncommented it?

Reply:
Yep, I had intended to do that after I ran it on CircleCI as well...just forgot. Gone now.